### PR TITLE
Updated run, crouch, prone, light, bleeding overlay icons mod for retrodays and neodays

### PR DESCRIPTION
**Pull request by dranitski**
#### Summary
Interface "Updated run, crouch, prone, light, bleeding overlay icons mod for retrodays and neodays"

#### Purpose of change

1. I am often forgetting to turn off my smartphone screen light / candle before going out from the shelter. Retrodays miss any overlay that shows player tile is being lit
2. Added R (running), C (crouch), P (prone) symbols on top of player to represent current move style 

#### Describe the solution

New icons dont swap on player tile flip.

**Bleeding:**

**Flashlight / Candle / Smartphone / Any other activated light source:**

**Prone**

**Bleeding, running and active light** 

#### Describe alternatives you've considered

.

#### Testing

Checked bleed, run, walk, prone, light activities.
Checked this when player face left and right to ensure letters not swap.

#### Additional context

Maybe someone can tell me why we keep this overlay icons in separate mod? While all other tilesets have it merged into tileset iteself. Maybe @Fris0uman know something on that? Thanks in advance!

Also I want to mention that i;ve found a way to prevent overlay effect icons from flipping horisontally with player icon, the only thing needed was to change the tile type from 'fg' to 'bg':
```
        "tiles": [
          { "id": "overlay_run", "bg": 1 },
          { "id": "overlay_crouch", "bg": 2 },
          { "id": "overlay_effect_bleed", "bg": 3 },
          { "id": "overlay_prone", "bg": 4 }
        ]
```

### CODE REVIEW

**Comment by Wokko1** [comment]
Maybe the active light overlay could be flipped with the player since it represents a light source in player's hands?

**Comment by dranitski** [comment]
@Wokko1 ive just played NeoDays for a while and can say that is definitively a good idea. Will fix

**Comment by dranitski** [comment]
@Wokko1

File: data/mods/sees_player_retro/mod_tileset.json [hunk 1]
```diff
@@ -3,16 +3,36 @@
...
+      },
+      {
+        "file": "sees_player_retro.png",
+        "sprite_width": 10,
+        "sprite_height": 10,
+        "sprite_offset_x": 0,
+        "sprite_offset_y": -1,
+        "tiles": [
+          { "id": "overlay_effect_haslight", "fg": 5 }
         ]
```
Comment (kevingranade) [inline data/mods/sees_player_retro/mod_tileset.json:36]:
```suggestion
        "tiles": [ { "id": "overlay_effect_haslight", "fg": 5 } ]
```

**Comment by kevingranade** [comment]
The light sprite looks AMAZING.

**Comment by dranitski** [comment]
@kevingranade sorry, maybe you know why we keep this overlay icons in separate mod? While all other tilesets have it merged into tileset iteself. Can I move that from mod to RetroDays and NeoDays tilesets instead? Thanks in advance!

**Comment by kevingranade** [comment]
IIRC they are separate because of a perception that RetroDays players would want to be able to disable the overlays for a cleaner "purist sprite" look that does not have overlays, and adding them as a mod is the mechanism for doing so.

**Comment by anothersimulacrum** [comment]
I will say that I do not want overlays. Though we should probably have them enabled by default if possible.

**Comment by anothersimulacrum** [comment]
And as always, how do these look on a post-thresh ursine or other mutant that changes the sprite?

**Comment by dranitski** [comment]
@anothersimulacrum it doesnt look perfect. Just as well as the NeoDays items overlays :grinning: 

NeoDays, lizard post-thresh, wearing cloths and wielding knife, running
image
NeoDays, cattle post-thrash, running
image

RetroDays, lizard post-thresh, running
image
RetroDays, cattle post-thresh, running
image

You can notice how bad NeoDays item and cloth overlays looks on these post-thresh characters, so that is a part of bigger problem, which, I believe, is out of scope of this commit. Bigger post-thresh char sprites in RetroDays and NeoDays is a problem that should be solved in some other ways, as I see. That is because game cant handle new offsets when char sprites becomes larger (for example 20x20 after being 10x10). I cant see other mods to have bigger player sprites for post-thresh mutations, this is an issue only for NeoDays and RetroDays, which should have its own github issue, which can be created by anyone who cares.

**Pull request merged**